coreos/config/env/dev-libs/libnl

```diff
@@ -0,0 +1,2 @@
+INSTALL_MASK+=" /etc/libnl/ /etc/libnl/classid /etc/libnl/pktloc"
+PKG_INSTALL_MASK+=" /etc/libnl/ /etc/libnl/classid /etc/libnl/pktloc"
```

Comment:
Can you just add `/etc/libnl` to `profiles/coreos/targets/generic/make.defaults`?  It's better to keep these centralized so we don't lose track of them.